Shared/Samples/Select features in scene layer/SelectFeaturesInSceneLayerView.swift

```diff
@@ -0,0 +1,91 @@
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct SelectFeaturesInSceneLayerView: View {
+    /// The point on the screen where the user tapped.
+    @State private var tapPoint: CGPoint?
+    
+    /// The scene layer that is added on top of the scene
+    @State private var sceneLayer = {
+        ArcGISSceneLayer(url: .brestBuildingService)
+    }()
+    
+    /// A scene with topographic basemap and a 3D buildings layer.
+    @State private var scene: ArcGIS.Scene = {
+        // Creates a scene and sets an initial viewpoint.
+        let scene = Scene(basemapStyle: .arcGISTopographic)
+        let point = Point(x: -4.49779, y: 48.38282, z: 40, spatialReference: .wgs84)
+        // set an initial camera location
+        let camera = Camera(location: point, heading: 41.65, pitch: 71.2, roll: 0)
+        // set initial viewpoint to camera position at point
+        scene.initialViewpoint = Viewpoint(boundingGeometry: point, camera: camera)
+        // Creates a surface and adds an elevation source.
+        let surface = Surface()
+        surface.addElevationSource(ArcGISTiledElevationSource(url: .worldElevationService))
+        // Sets the surface to the scene's base surface.
+        scene.baseSurface = surface
+        return scene
+    }()
+    
+    /// Add feature layer to scene in initialization
+    init() {
+        scene.addOperationalLayer(sceneLayer)
+    }
+    
+    var body: some View {
+        // Wrap the SceneView in SceneViewReader and expose its SceneViewProxy.
+        SceneViewReader { sceneViewProxy in
+            SceneView(scene: scene)
+                // Capture location of tap on screen
+                .onSingleTapGesture { screenPoint, _ in
+                    tapPoint = screenPoint
+                }
+                .task(id: tapPoint) {
+                    // clear previous selections
+                    sceneLayer.clearSelection()
+                    do {
+                        // ensure there is a tap point captures
+                        guard let locationPoint = tapPoint else { return }
+                        // get a list of possible of the elements that were tapped
+                        let result = try await sceneViewProxy.identify(on: sceneLayer, screenPoint: locationPoint, tolerance: 10)
+                        // grab the first selected element in the array
+                        guard let feature = result.geoElements.first as? ArcGISFeature else { return }
+                        // highlight the selected feature
+                        sceneLayer.selectFeature(feature)
+                    } catch {
+                       return
+                    }
```

Comment:
See https://github.com/Esri/arcgis-maps-sdk-swift-samples/blob/main/Shared/Samples/Select%20features%20in%20feature%20layer/SelectFeaturesInFeatureLayerView.swift as an example on how errors are handled.

Reply:
Gotcha, I think it handles correctly. When tested with random error it shows error popup now.